### PR TITLE
update optimal core allocation experiment results (release v0.12)

### PR DESCRIPTION
**Pull request by ppalucki**
Fixes issue "use new updated configuration and binary for gathering experiment results"

Summary of changes:
- just update new images moved to new location
- minimal updates in conclussions (we have now 10 load points up ti 1.5 m QPS, so reasults are slightly differnt

Testing done:
- check a README [here ](https://github.com/intelsdi-x/swan/tree/ppalucki/update-core-results/experiments/optimal-core-allocation) (scroll down)

### CODE REVIEW

**Comment by ppalucki** [comment]
run integration tests

**Comment by ppalucki** [comment]
run integration tests

File: experiments/optimal-core-allocation/README.md [hunk 1]
```diff
@@ -51,10 +51,12 @@ Note that you need to specify your latency SLO in micro seconds to mark a config
...
-from generic import optimal_core_allocation_qps, optimal_core_allocation_latency, optimal_core_allocation_cpu, 
-optimal_core_allocation_qps("ca24aa5d-4a88-7258-6f00-5f651b0d6515", slo=500)
-optimal_core_allocation_latency("ca24aa5d-4a88-7258-6f00-5f651b0d6515")
-optimal_core_allocation_cpu("ca24aa5d-4a88-7258-6f00-5f651b0d6515") # only available when run with -use_USE_collector flag
+from swan import Experiment, OptimalCoreAllocation
+exp = Experiment("ca24aa5d-4a88-7258-6f00-5f651b0d6515")
+core = OptimalCoreAllocation(exp, slo=500)
+core.latency()
+core.qps()
+core.cpu() # if run with USE snap collector
```
Comment (skonefal) [inline experiments/optimal-core-allocation/README.md]: 'Requires Snap USE collector'

**Comment by ppalucki** [comment]
run integration tests

**Pull request merged**